programs/programs/urgent_needs/ma/free_tax_help.py

```
@@ -0,0 +1,25 @@
+from ..base import UrgentNeedFunction
+
+
+class FreeTaxHelp(UrgentNeedFunction):
+    """
+    CEOC – Free Tax Help (VITA)
```

Ruff (0.14.8)

Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)